cli/src/vote.rs

```diff
@@ -117,6 +142,39 @@ pub fn process_authorize_voter(
     log_instruction_custom_error::<VoteError>(result)
 }
 
+pub fn process_authorize_withdrawer(
```

Comment:
What do you think about some sort of VoteAuthorized enum with Voter and Withdrawer variants to pass into process (and a generic vote_instruction::authorize, etc), so that we don't need as much duplication?
Or do you anticipate the needs/functionality of authorize_voter and authorize_withdrawer diverging in the future?

Reply:
that sounds like a good idea...  the replicode bothered me, too